scripts/generate_chain_endpoints_mapping.py

```diff
@@ -0,0 +1,244 @@
+import asyncio
+import json
+import time
+
+from aiohttp import ClientSession
+from pathlib import Path
+from typing import Dict, List, Tuple
+from urllib.parse import urlparse
+
+
+#
+# Only LYNX for now
+#
+
+#MAINNET_CHAINS = [
+#    1,  # Ethereum Mainnet
+#    137  # Polygon Mainnet
+#]
+
+#TAPIR_CHAINS = [
+#    80002,  # Polygon Amoy
+#    11155111  # Sepolia
+#]
+
+LYNX_CHAINS = {
+    1,  # Ethereum Mainnet
+    10,  # OP Mainnet
+    56,  # BNB Smart Chain Mainnet
+    97,  # BNB Smart Chain Testnet
+    100,  # Gnosis
+    137,  # Polygon Mainnet
+    300,  # zkSync Sepolia Testnet
+    314,  # Filecoin - Mainnet
+    397,  # NEAR Protocol
+    398,  # NEAR Protocol Testnet
+    7700,  # Canto
+    7701,  # Canto Tesnet
+    8453,  # Base
+    10200,  # Gnosis Chiado Testnet
+    42161,  # Arbitrum One
+    42220,  # Celo Mainnet
+    43114,  # Avalanche C-Chain
+    80002,  # Amoy
+    84532,  # Base Sepolia Testnet
+    421614,  # Arbitrum Sepolia
+    534351,  # Scroll Sepolia Testnet
+    534352,  # Scroll
+    3141592,  # Filecoin - Butterfly testnet
+    11155111,  # Sepolia
+    11155420,  # OP Sepolia Testnet
+}
+
+EXTRA_KNOWN_RPC_ENDPOINTS = {
```

Comment:
love that you'd added drpc endpoints here. Similary I think we should add some from https://www.nodies.app/ which is built on top of POKT (similar to dRPC) - although, as I type this, I can't remember if pokt endpoints are compatible with TACo...

`https://ethereum-sepolia-public.nodies.app`

Reply:
✔️

Reply:
> I can't remember if pokt endpoints are compatible with TACo...

There was an issue before, but it was fixed via https://github.com/nucypher/nucypher/pull/3491 for nodes to be agnostic to RPC endpoints.